### PR TITLE
fix: verify-token

### PR DESCRIPTION
**Pull request by laurentsimon**
Successful run https://github.com/laurentsimon/slsa-github-generator/actions/runs/4407749051

### CODE REVIEW

File: .github/workflows/scripts/e2e-verify.common.sh [hunk 1]
```diff
@@ -67,8 +67,10 @@ e2e_verify_common_all_v1() {
 # Verifies common fields of the SLSA v1.0 predicate buildDefinition.
 # $1: the predicate content
 e2e_verify_common_buildDefinition_v1() {
+    echo "pred: $1"
     # This does not include buildType since it is not common to all.
     e2e_verify_predicate_v1_buildDefinition_externalParameters_source "$1" "{\"uri\":\"git+https://github.com/$GITHUB_REPOSITORY@$GITHUB_REF\",\"digest\":{\"sha1\":\"$GITHUB_SHA\"}}"
+    e2e_verify_predicate_v1_buildDefinition_externalParameters_inputs "$1" '{"name1":"value1","name2":"***","name3":"value3","name4":"***","name5":"value5","name6":"***","private-repository":true}'
```
Comment (asraa) [inline .github/workflows/scripts/e2e-verify.common.sh:73]: This is common build definition verification for all v1.0 provenance - the docker-based ones don't include inputs like this, so it'll make that job fail

I'd move this line to the verify-setup-token.sh / verify-verified-token.sh scripts

Comment (laurentsimon) [reply .github/workflows/scripts/e2e-verify.common.sh:73]: verify-setup-token does not mask inputs. I'll move this line to the verify-verified-token.sh instead. Works?

**Review by asraa — APPROVED** [review]
Yes, looks good now!

File: .github/workflows/scripts/schedule.actions/verify-verified-token.sh [hunk 2]
```diff
@@ -21,6 +21,8 @@ echo "PREDICATE_CONTENT: $PREDICATE_CONTENT"
 
 # Verify common predicate fields.
 e2e_verify_common_all_v1 "$PREDICATE_CONTENT"
-e2e_verify_predicate_v1_buildDefinition_buildType "$PREDICATE_CONTENT" "https://github.com/slsa-framework/slsa-github-generator/delegator-generic@v0"
+e2e_verify_predicate_v1_buildDefinition_buildType "$PREDICATE_CONTENT" "https://github.com/slsa-framework/slsa-github-generator/delegator-generic@v1"
```
Comment (ianlewis) [inline .github/workflows/scripts/schedule.actions/verify-verified-token.sh:24]: I somehow lost my review comment fixes on #1773. I think this should have been changed back to 'v0' based on the comments there. I'll fix this.

**Pull request merged**